### PR TITLE
Building a Go Web App with Gin, MongoDB, and Cody

### PR DESCRIPTION
**Pull request by kukicado**
Blog post was originally published on MDB developer center, but we can republish it here. 

We'll ship this post on Sunday.

### CODE REVIEW

File: content/blogposts/2023/building-a-go-web-app-with-gin-mongodb-cody-ai.md [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+Next, initialize a new [Go module](https://go.dev/blog/using-go-modules), which will manage dependencies for our project:
+
+```bash
+go mod init mflix
+```
+
+Now that we have our Go module created, let's install the dependencies for our project. We'll keep it really simple and just install the gin and mongodb libraries.
+
+```bash
+go  get  github.com/gin-gonic/gin
```
Comment (jdorfman) [inline content/blogposts/2023/building-a-go-web-app-with-gin-mongodb-cody-ai.md:54]: Remove double space

File: content/blogposts/2023/building-a-go-web-app-with-gin-mongodb-cody-ai.md [hunk 2]
```diff
@@ -0,0 +1,609 @@
...
+package main
+
+import (
+    // Add required Go packages
+    "context"
+    "log"
+
+    "github.com/gin-gonic/gin"
+
+       // Add the MongoDB driver packages
```
Comment (jdorfman) [inline content/blogposts/2023/building-a-go-web-app-with-gin-mongodb-cody-ai.md:109]: Align comment with "go.mongodb..."

**Review by jdorfman — CHANGES_REQUESTED** [review]

File: content/blogposts/2023/building-a-go-web-app-with-gin-mongodb-cody-ai.md (hunk 1 above)
Comment (jdorfman) [reply content/blogposts/2023/building-a-go-web-app-with-gin-mongodb-cody-ai.md:54]: `go  get  github.com/gin-gonic/gin`

`go get github.com/gin-gonic/gin`

**Review by jdorfman — APPROVED** [review]
Looks good, we just need to look into the disappearing syntax highlighting bug https://www.loom.com/share/b89b5f26a16947319f1a165c595f8a17

**Comment by kukicado** [comment]
For sure. I'll merge this PR. We def need to give the code blocks some design/functionality love. It's not a good experience at the moment.

**Pull request merged**